aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/EventBridgeEvent.java

```diff
@@ -0,0 +1,31 @@
+package com.amazonaws.services.lambda.runtime.events;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+/**
+ * Class to represent the EventBridge event. This is also the CloudWatch Events event format.
+ *
+ * @see <a href="https://docs.aws.amazon.com/lambda/latest/dg/services-cloudwatchevents.html">CloudWatch Events</a>
+ */
+
+@Data
+@Builder(setterPrefix = "with")
+@NoArgsConstructor
+@AllArgsConstructor
+public class EventBridgeEvent {
```

Comment:
Using the EventBridge "Download code bindings" feature actually provides an AWSEvent (below). Is it useful to have it in the lib, and if so, why not simply use the generic one below?

```
public class AWSEvent<T> {

  @JsonProperty("detail")
  private T detail = null;

  @JsonProperty("detail-type")
  private String detailType = null;

  @JsonProperty("resources")
  private List resources = null;

  @JsonProperty("id")
  private String id = null;

  @JsonProperty("source")
  private String source = null;

  @JsonProperty("time")
  private Date time = null;

  @JsonProperty("region")
  private String region = null;

  @JsonProperty("version")
  private String version = null;

  @JsonProperty("account")
  private String account = null;

// ...
```

Reply:
I don't think the Lambda service would be able to handle the typing.

@carlzogh

Reply:
Unfortunately not at this point, the Java runtime needs to handle `@JsonProperty` annotations internally